linux_os/guide/system/permissions/files/sysctl_fs_protected_hardlinks/rule.yml

```diff
@@ -21,6 +21,7 @@ references:
     anssi: NT28(R23)
     cis: 1.6.1
     nist: SI-11
+    srg: SRG-OS-000324-GPOS-00125
```

Comment:
Ack

ref: 

SRG-OS-000324-GPOS-00125 |   | The operating system must prevent non-privileged users from executing privileged functions to include disabling, circumventing, or altering implemented security safeguards/countermeasures.
-- | -- | --